internal/backend/workflows/common.go

```diff
@@ -0,0 +1,141 @@
+package workflows
```

Comment:
`commonsessionworkflow`?

Reply:
too long for me and IMO it's reduces readability :(

Reply:
I would leave the naming as is - it's just the common functions/things used in all our workflows